### PR TITLE
Add JS202011SCQN DPDT Slide Switch

### PR DESCRIPTION
**Pull request by dbrgn**
image

https://www.digikey.ch/product-detail/en/c-k/JS202011SCQN/401-2002-1-ND/1640098.

Based on https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/36, merge that first.

### CODE REVIEW

**Comment by ubruhin** [comment]
@dbrgn since https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/36 is merged, this one is ready for review, right? Can you rebase to fix the merge conflict?

**Comment by dbrgn** [comment]
@ubruhin rebased, ready for review!

File: pkg/420f1a95-2058-4f30-bf54-3d06eb49b2e3/package.lp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center bottom) (position 0.0 5.225) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{NAME}}")
+  )
+  (stroke_text be3110e9-e6fc-454c-8721-eec042288076 (layer top_values)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center top) (position 0.0 -5.225) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{VALUE}}")
+  )
+  (stroke_text 86e43e68-ddf7-4a2c-a276-3e1d72be329d (layer brd_documentation)
```
Comment (ubruhin) [inline pkg/420f1a95-2058-4f30-bf54-3d06eb49b2e3/package.lp]: The text should be on top documentation:

```suggestion
  (stroke_text 86e43e68-ddf7-4a2c-a276-3e1d72be329d (layer top_documentation)
```

**Review by ubruhin — CHANGES_REQUESTED** [review]
Looks very good, except the layer of the "C&K" text :)

File: pkg/420f1a95-2058-4f30-bf54-3d06eb49b2e3/package.lp (hunk 1 above)
Comment (dbrgn) [reply pkg/420f1a95-2058-4f30-bf54-3d06eb49b2e3/package.lp]: Fixed!

**Review by ubruhin — APPROVED** [review]

**Pull request merged**